### PR TITLE
MEN-6821: documentation to set up SAML and AWS IAM Identity Center

### PR DESCRIPTION
**Pull request by tranchitella**

### CODE REVIEW

File: 08.Server-integration/08.SAML-Federated-Authentication/01.AWS-IAM-Identity-Center/docs.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+* Choose a Display Name (e.g., Hosted Mender) and a description for your users;
+* Download the IAM Identity Center SAML metadata file clicking on the "Download" link.
+
+![Download the SAML metadata](03-add-application-download-saml-metadata.png)∏
+
+
+### Configure Mender as a Service Provider
+
+Open your Hosted Mender account in another browser tab to set it up as a SAML 2.0 Service Provider (SP).
```
[the remark this answers is not in the document]
Comment (tranchitella) [reply 08.Server-integration/08.SAML-Federated-Authentication/01.AWS-IAM-Identity-Center/docs.md:44]: I also replaced `Hosted Mender` with `hosted Mender` everywhere in this page

File: 08.Server-integration/08.SAML-Federated-Authentication/01.AWS-IAM-Identity-Center/docs.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+We then scroll down to the "IAM Identity Center metadata" section and click on the "Download" link for the "IAM Identity Center SAML metadata file" field.
+
+The newly downloaded XML will have the following line in it:
+
+```xml
+<md:NameIDFormat>urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress</md:NameIDFormat>
+```
+
+Lastly, we can proceed to replace the SAML metadata file in Hosted Mender in the "Organization and billing" settings view, as explained above.
+Please note that the Mender's SAML values (Start URL, ACS URL, Entity ID) won't change after you successfully upload the new SAML metadata file.
```
[the remark this answers is not in the document]
Comment (tranchitella) [reply 08.Server-integration/08.SAML-Federated-Authentication/01.AWS-IAM-Identity-Center/docs.md:93]: Don't know, the UI doesn't show the URLs upfront. I suppose we never completed that task.

File: 08.Server-integration/08.SAML-Federated-Authentication/01.AWS-IAM-Identity-Center/docs.md [hunk 3]
```diff
@@ -8,13 +8,13 @@ taxonomy:
...
 
-This example will show you how to set AWS IAM Identity Center as an Identity Provider (IdP) for Hosted Mender as a Service Provider (SP).
+This example will show you how to set AWS IAM Identity Center as an Identity Provider (IdP) for hosted Mender as a Service Provider (SP).
 
 
 ## Prerequisites
 
-* Hosted Mender on a [free trial](https://mender.io/demo) or Enterprise plan.
-* An AWS account with enough privileges to create a customer managed SAML 2.0 application in the IAM Identity Center
+* Hosted Mender Enterprise or a [free trial](https://mender.io/demo);
```
Comment (merlin-northern) [inline 08.Server-integration/08.SAML-Federated-Authentication/01.AWS-IAM-Identity-Center/docs.md:16]: this semicolon at the end, on purpose?

**Review by merlin-northern — APPROVED** [review]
looks good to me, one nit.

File: 08.Server-integration/08.SAML-Federated-Authentication/01.AWS-IAM-Identity-Center/docs.md (hunk 3 above)
Comment (tranchitella) [reply 08.Server-integration/08.SAML-Federated-Authentication/01.AWS-IAM-Identity-Center/docs.md:16]: Yes, I usually do semicolon at the end of each bullet point but the last one, which has a period.

**Pull request merged**